Jenkinsfile

```diff
@@ -10,20 +10,35 @@
 import groovy.json.JsonOutput
 import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 
-// As always with modified properties.parameters below, mind we need a 2nd job
-// run to get them loaded[1], fyi this 2nd run can be triggered by adding a PR
-// comment with a `bors try` line.
+// Workaround https://issues.jenkins-ci.org/browse/JENKINS-41929 (which forces
+// a 2nd job run to get them loaded[1], this 2nd run can be triggered by
+// adding a PR comment with a `bors try` line): override buildRelDefaults if
+// corresponding parameter is set.
 // [1] see: https://issues.jenkins-ci.org/browse/JENKINS-41929
+
+def buildRelDefaults = [
+  'AKS_REL': '1.15,1.16',
+  'EKS_REL': '1.15,1.16',
+  'GKE_REL': '1.15,1.16',
+  'GEN_REL': '1.16',
+]
+
 properties([
   // See releasesFromStr() function below on how we parse the _REL string
   parameters([
-    stringParam(name: 'AKS_REL', defaultValue: '1.15,1.16', description: 'AKS releases to test (comma separated)'),
-    stringParam(name: 'EKS_REL', defaultValue: '1.14,1.15', description: 'EKS releases to test (comma separated)'),
-    stringParam(name: 'GKE_REL', defaultValue: '1.15,1.16-pre', description: 'GKE releases to test (comma separated)'),
-    stringParam(name: 'GEN_REL', defaultValue: '1.15', description: 'Generic-cloud releases to test (comma separated)'),
+    stringParam(name: 'AKS_REL', defaultValue: '', description: "Override AKS releases to test"),
+    stringParam(name: 'EKS_REL', defaultValue: '', description: "Override EKS releases to test"),
+    stringParam(name: 'GKE_REL', defaultValue: '', description: "Override GKE releases to test"),
+    stringParam(name: 'GEN_REL', defaultValue: '', description: "Override Generic-cloud releases to test"),
+    stringParam(name: 'TIMEOUT', defaultValue: '150', description: 'Full e2e tests run timeout in minutes'),
+    stringParam(name: 'PAUSE_TIME', defaultValue: '15', description: 'Time to pause on failure in minutes (before deletion)'),
```

Comment:
re-lgtm, thanks for the diff.

Ooc, which is the use case of the releases overrides?

Reply:
> Ooc, which is the use case of the releases overrides?

E.g. manually testing a new release, or looping on an specific one (ie removing others) while debugging (this latter was my use case, focusing on `generic` build issues we had).